docs/src/community/phrasebook.rst

```diff
@@ -0,0 +1,63 @@
+.. include:: ../common_links.inc
+
+.. _phrasebook:
+
+Package Phrasebook
+===================
+
+There are a number of similar packages to Iris, and a lot of these have their own
+terminology for similar things. Whether you're coming or going, we hope this might
+be a helpful guide to these differences!
+Definitions for each can be found in :ref:`glossary`. See also
+`Xarray terminology <https://docs.xarray.dev/en/stable/user-guide/terminology.html>`_.
+
+.. list-table:: Phrasebook
+   :widths: 25 25 25 50
+   :header-rows: 1
+
+   * - Iris
+     - Xarray
+     - Example
+     - Notes
+   * - Non-Lazy
+     - Eager
+     -
+     - Used to relate to functions, rather than the data.
+   * - Cube
+     - DataArray
+     -
+     -
+   * - Cubelist
+     - Dataset
+     -
+     -
+   * - Merge
+     - Concatenate
+     -
+     - Functionality is essentially swapped.
+   * - Concatenate
+     - Merge
+     -
+     - Functionality is essentially swapped.
+   * - Scalar Coordinate
+     - Scalar
+     -
+     - Coordinates in Iris and Xarray are split via rules, and so are not a
```

Comment:
I'm not entirely clear what "split via rules" means in this context, could this be clarified?